jest.config.js

```diff
@@ -26,4 +26,5 @@ module.exports = {
   transform: {
     '^.+\\.[t|j]sx?$': 'babel-jest',
   },
+  snapshotSerializers: [],
```

Comment:
why is this needed?

Reply:
This was added during the RTL migration. https://github.com/americanexpress/holocron/pull/166